### ISSUE TITLE
Better Indicator for Romeo's Special Ending

### ISSUE THREAD

**Issue by qbitrichard** [nospoone/holobunnies-issues#92]
It would be nice if it was more apparent when Romeo's ability is ending. Maybe turn the flashing red to just a regular flashing during final second or two so player knows it is about to end.

**Comment by nospoone** [comment]
Yeah, Claude and I had been talking about adding something like this. I will probably make a bar over the bunny's head to indicate the time remaining.

**Comment by nospoone** [comment]
[Added it here](https://trello.com/c/8Ka3gc7V/3-boss-rush).
